docs/translating/spanish.md

```diff
@@ -34,10 +34,10 @@ Zulip friendly and usable.
 ## Términos
 
 - Message - **Mensaje**
-- Private message (PM) - **Mensaje privado (MP)**
-- Group PM - **mensaje privado grupal**: even though "MP grupal" is the most
+- Direct message (DM) - **Mensaje directo (MD)**
+- Group PM - **mensaje directo grupal**: even though "MD grupal" is the most
```

Comment:
This needs to be "Group DM - "

Reply:
Not sure how I missed this 😅.  Glad you caught it!